api/envoy/config/core/v3/health_check.proto

```diff
@@ -145,6 +145,11 @@ message HealthCheck {
     // <envoy_v3_api_msg_type.matcher.v3.StringMatcher>`. See the :ref:`architecture overview
     // <arch_overview_health_checking_identity>` for more information.
     type.matcher.v3.StringMatcher service_name_matcher = 11;
+
+    // An optional string to match anywhere in the first 1024 bytes of the response body.
+    // The response data can only be ASCII. If it is set, both the expected response check and status code determine the health check.
+    google.protobuf.StringValue response = 13
```

Comment:
I do not think these constraints are correct as the body can have arbitrary characters. Also the type should be `bytes`, not `string` as the protobuf `string` implies UTF-8 encoding.

Reply:
Thank you for reviewing! Relaxed the limitation, and changed to bytes.